graspologic/layouts/auto.py

```diff
@@ -71,6 +72,8 @@ def layout_tsne(
         ``max_edges`` exist. Warning: this pruning is approximate and more edges than
         are necessary may be pruned. Running in 32 bit enviornment you will most
         likely need to reduce this number or you will out of memory.
+    random_seed : int
+        Seed to be used for reproducable results. Default is None.
```

Comment:
I think it would be good to add that `None` will produce a random state. It is generally common knowledge but being explicit is preferred :)